.github/workflows/slack-notification.yml

```diff
@@ -9,21 +9,29 @@ on:
     types:
       - completed
 jobs:
+  get-branch:
+    name: Get Current Branch Name
+    runs-on: ubuntu-latest
+    outputs:
+      branch: ${{ steps.extract_branch.outputs.branch }}
+    if: ${{ github.event.workflow_run.conclusion != 'failure' }}
+    steps:
+      - name: Extract branch name
+        id: extract_branch
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
```

Comment:
In the linked logs, I see that the echoed stmt is this same string as is "echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})", is it expected?